CMakeLists.txt

```diff
@@ -106,10 +106,10 @@ message(STATUS "png: ${OPTION_PNG}")
 set(OPTION_UNISCRIBE ${AUTODETECT_UNISCRIBE} CACHE BOOL "Use uniscribe support")
 message(STATUS "Uniscribe: ${OPTION_UNISCRIBE}")
 
-set(OPTION_SSE ${AUTODETECT_SSE} CACHE BOOL "Use SSE support")
-message(STATUS "SSE: ${OPTION_SSE}")
-set(OPTION_HAVE_THREAD YES CACHE BOOL "Use threads")
-message(STATUS "Threads: ${OPTION_HAVE_THREAD}")
+set(OPTION_USE_SSE ${AUTODETECT_SSE} CACHE BOOL "Use SSE support")
+message(STATUS "SSE: ${OPTION_USE_SSE}")
+set(OPTION_USE_THREADS YES CACHE BOOL "Use threads")
```

Comment:
These SSE and HAVE_THREAD changes require update in source.list else parsing fails

Reply:
Oops :D Tnx!